src/helpers/NameHelpers.ts

```diff
@@ -4,7 +4,8 @@ import { CommonModel } from 'models';
  * Default property names for different aspects of the common model
  */
 export enum DefaultPropertyNames {
-  additionalProperties = 'additionalProperties'
+  additionalProperties = 'additionalProperties',
+  patternProperty = 'PatternProperty'
```

Comment:
```suggestion
  patternProperty = 'patternProperties'
```

Because we save pattern properties as collection, not single one :)

Reply:
That's true, but each pattern is rendered as property here, meaning the default property name for pattern properties are `{pattern}{DefaultPropertyNames.patternProperty}`. 

To me this makes more sense then using plural 🤔?

Reply:
I guess you are right, changing 🧐